R/derive_vars_dy.R

```diff
@@ -44,37 +44,43 @@
 #'
 #' datain <- tibble::tribble(
 #'   ~TRTSDTM, ~ASTDTM, ~AENDT,
-#'  "2014-01-17T23:59:59", "2014-01-18T13:09:O9", "2014-01-20"
+#'   "2014-01-17T23:59:59", "2014-01-18T13:09:O9", "2014-01-20"
 #' ) %>%
-#'  mutate(TRTSDTM = as_datetime(TRTSDTM),
-#'         ASTDTM = as_datetime(ASTDTM),
-#'         AENDT = ymd(AENDT))
+#'   mutate(
+#'     TRTSDTM = as_datetime(TRTSDTM),
+#'     ASTDTM = as_datetime(ASTDTM),
+#'     AENDT = ymd(AENDT)
+#'   )
 #'
 #' derive_vars_dy(
 #'   datain,
 #'   reference_date = TRTSDTM,
-#'   source_vars = vars(TRTSDTM, ASTDTM, AENDT))
+#'   source_vars = vars(TRTSDTM, ASTDTM, AENDT)
+#' )
 #'
 #' # specifying name of new variables
 #' datain <- tibble::tribble(
 #'   ~TRTSDT, ~DTHDT,
-#'  "2014-01-17", "2014-02-01"
+#'   "2014-01-17", "2014-02-01"
 #' ) %>%
-#'  mutate(TRTSDT = ymd(TRTSDT),
-#'         DTHDT = ymd(DTHDT))
+#'   mutate(
+#'     TRTSDT = ymd(TRTSDT),
+#'     DTHDT = ymd(DTHDT)
+#'   )
 #'
 #' derive_vars_dy(datain,
```

Comment:
Should `datain` on line 71 be on next line to keep function parameters aligned.

Reply:
Yes